statsd/options.go

```diff
@@ -0,0 +1,109 @@
+package statsd
+
+import "time"
+
+var (
+	// DefaultNamespace is the default value for the Namespace option
+	DefaultNamespace = ""
+	// DefaultTags is the default value for the Tags option
+	DefaultTags = []string{}
+	// DefaultBuffered is the default value for the Buffered option
+	DefaultBuffered = false
+	// DefaultMaxMessagesPerPayload is the default value for the MaxMessagesPerPayload option
+	DefaultMaxMessagesPerPayload = 16
+	// DefaultBlockingUDS is the default value for the BlockingUDS option
+	DefaultBlockingUDS = false
```

Comment:
Blocking UDS the default behavior earlier, so this would change the behavior for customer clients possibly unwillingly. I personally would love the gains, but maybe the buffered UDS sender should be an opt-in feature?

Reply:
Let's keep blocking in 2.x